t/full-stack.t

```diff
@@ -243,6 +244,32 @@ like(
     'test 2 is restarted by killing worker'
 );
 
+$driver->find_element_by_id('cancel_running')->click();
```

Comment:
This sounds like a bug to me. The job is incomplete and cloned as 3. What are we cancelling?